app/controllers/provider_interface/interviews_controller.rb

```diff
@@ -0,0 +1,67 @@
+module ProviderInterface
+  class InterviewsController < ProviderInterfaceController
+    before_action :set_application_choice
+    before_action :interview_flag_enabled?
+    before_action :requires_make_decisions_permission
+
+    def index
+      @provider_can_respond = current_provider_user.authorisation.can_make_decisions?(
+        application_choice: @application_choice,
+        course_option_id: @application_choice.offered_option.id,
+      )
+
+      @interviews = @application_choice.interviews.kept.includes([:provider])
+    end
+
+    def new
+      CreateInterview.new(
+        actor: current_provider_user,
+        application_choice: @application_choice,
+        provider: @application_choice.offered_course.provider,
+        date_and_time: Time.zone.now,
+        location: 'Zoom',
+        additional_details: 'Some preferences',
+      ).save!
```

Comment:
To be removed. As it enables testing on the review app so will leave for now.

Reply:
How about we introduce a `before_action` for the entire controller requiring the feature flag to be on? Along the lines of https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/controllers/provider_interface/reasons_for_rejection_controller.rb#L3

Then we don't need to worry about what `new` or any other action does etc.

Reply:
There i a feature flag. I just dont want to merge anything which hardcodes informatino (even though I am aware that we should never get here unless the flag is enabled).

Reply:
In that case, great!